adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/GenotypeRDD.scala

```diff
@@ -159,17 +162,39 @@ case class GenotypeRDD(rdd: RDD[Genotype],
   /**
    * Explicitly saves to VCF.
    *
+   * @param args Arguments defining where to save the file.
+   * @param stringency The validation stringency to use when writing the VCF.
+   *   Defaults to LENIENT.
+   */
+  def saveAsVcf(args: ADAMSaveAnyArgs,
```

Comment:
I think this is inconsistent with the direction we've been going, where we've broken up `GenomicRDD` methods that transiently change the underlying type of the `GenomicRDD` (e.g., `toCoverage(collapse = true)` becoming `toCoverage().collapse()`. I think that this method is particularly problematic because the changes to the method actually break the utility of the method. Specifically, this change means that the `.save` method always saves unsorted VCF, which is non-compliant with the VCF spec.